app/templates/request/responses/modal_body/notes.html

```diff
@@ -1,9 +1,21 @@
+<div>
+    Created By:
+    {% if current_user.is_agency %}
```

Comment:
I think this should be swapped with the statement below. That way if an agency user created the comment, they see you, otherwise they see the name of the user who created it.